packages/deployments/contracts/deploy/03_deployConnext.ts

```diff
@@ -241,6 +246,7 @@ const func: DeployFunction = async (hre: HardhatRuntimeEnvironment): Promise<voi
     { name: getDeploymentName("SwapAdminFacet"), contract: "SwapAdminFacet", args: [] },
     { name: getDeploymentName("VersionFacet"), contract: "VersionFacet", args: [] },
     { name: getDeploymentName("DiamondCutFacet"), contract: "DiamondCutFacet", args: [] },
+    { name: getDeploymentName("DiamondLoupeFacet"), contract: "DiamondLoupeFacet", args: [] },
   ];
```

Comment:
Why should we deploy `DiamondLoupeFacet` manually? I believe that `hardhat-deploy` deploys that.
Currently this is causing `function "facetAddress" will shadow "facetAddress". Please update code to avoid conflict.` error while deploying.